unicode_smilies/unicode_smilies.php

```diff
@@ -257,4 +264,1766 @@ function unicode_smilies_smilies(&$a,&$b) {
+	Smilies::add($b, ':circled M', '&#x24C2');
+	Smilies::add($b, ':NEW button', '&#x1F195');
+	Smilies::add($b, ':NG button', '&#x1F196');
+	Smilies::add($b, ':O button (blood type)', '&#x1F17E');
+	Smilies::add($b, ':OK button', '&#x1F197');
+	Smilies::add($b, ':P button', '&#x1F17F');
+	Smilies::add($b, ':SOS button', '&#x1F198');  
+	Smilies::add($b, ':UP! button', '&#x1F199');
+	Smilies::add($b, ':VS button', '&#x1F19A');
+	Smilies::add($b, ':Japanese “here” button', '&#x1F201');
+	Smilies::add($b, ':Japanese “service charge” button', '&#x1F202');
+	Smilies::add($b, ':Japanese “monthly amount” button', '&#x1F237');
+	Smilies::add($b, ':Japanese “not free of charge” button', '&#x1F236');
+	Smilies::add($b, ':Japanese “reserved” button', '&#x1F22F');
+	Smilies::add($b, ':Japanese “bargain” button', '&#x1F250');
+	Smilies::add($b, ':Japanese “discount” button', '&#x1F239');
+	Smilies::add($b, ':Japanese “free of charge” button', '&#x1F21A');
+	Smilies::add($b, ':Japanese “prohibited” button', '&#x1F232');
+	Smilies::add($b, ':Japanese “acceptable” button', '&#x1F251');
+	Smilies::add($b, ':Japanese “application” button', '&#x1F238');
+	Smilies::add($b, ':Japanese “passing grade” button', '&#x1F234');
+	Smilies::add($b, ':Japanese “vacancy” button', '&#x1F233');
+	Smilies::add($b, ':Japanese “congratulations” button', '&#x3297');
+	Smilies::add($b, ':Japanese “secret” button', '&#x3299');
+	Smilies::add($b, ':Japanese “open for business” button', '&#x1F23A');
+	Smilies::add($b, ':Japanese “no vacancy” button', '&#x1F235');
+
+// geometric
+	Smilies::add($b, ':red circle', '&#x1F534');
+//	Smilies::add($b, ':orange circle', '&#x1F7E0');
+//	Smilies::add($b, ':yellow circle', '&#x1F7E1');
+//	Smilies::add($b, ':green circle', '&#x1F7E2');
+	Smilies::add($b, ':blue circle', '&#x1F535');
+//	Smilies::add($b, ':purple circle', '&#x1F7E3');
+//	Smilies::add($b, ':brown circle', '&#x1F7E4');
+	Smilies::add($b, ':black circle', '&#x26AB');
+	Smilies::add($b, ':white circle', '&#x26AA');
+//	Smilies::add($b, ':red square', '&#x1F7E5');
+//	Smilies::add($b, ':orange square', '&#x1F7E7');
+//	Smilies::add($b, ':yellow square', '&#x1F7E8');
+//	Smilies::add($b, ':green square', '&#x1F7E9');
+//	Smilies::add($b, ':blue square', '&#x1F7E6');
+//	Smilies::add($b, ':purple square', '&#x1F7EA');
+//	Smilies::add($b, ':brown square', '&#x1F7EB');
+	Smilies::add($b, ':black large square', '&#x2B1B');
+	Smilies::add($b, ':white large square', '&#x2B1C');
+	Smilies::add($b, ':black medium square', '&#x25FC');
+	Smilies::add($b, ':white medium square', '&#x25FB');
+	Smilies::add($b, ':black medium-small square', '&#x25FE');
+	Smilies::add($b, ':white medium-small square', '&#x25FD');
+	Smilies::add($b, ':black small square', '&#x25AA');
+	Smilies::add($b, ':white small square', '&#x25AB');
+	Smilies::add($b, ':large orange diamond', '&#x1F536');
+	Smilies::add($b, ':large blue diamond', '&#x1F537');
+	Smilies::add($b, ':small orange diamond', 
```

Comment:
Why didn't you use the semicolon syntax for this?

Reply:
Since I'm not a developer and therefore I followed the guidelines of @annando  Programmatically, after I saw that it works ;)

Reply:
Ok, please use `:wales` instead.